### ISSUE TITLE
[Request] Add ImageMagick

### ISSUE THREAD

**Issue by ruudvh** [zach-adams/hgv-deploy-full#5]
Considering this is a Wordpress setup I would recommend installing Imagick when PHP is installed.
ImageMagick provides better quality images (granted at the expensive of bigger files) and performance compared to the standard GD Library.

Wordpress itself searches [first](https://github.com/WordPress/WordPress/blob/efe36dbd5f78937ea4892a1bdf20b802cec80944/wp-includes/media.php#L2368) for ImageMagick, if it can't find it it will fall back to the GD Libary.

`php-fpm/tasks/main.yml`

``` yml
name: Install PHP packages
run_once: true
apt: name={{ item }} state=present
with_items:
- php5
- php5-cli
- php5-gd
- php5-imagick
- php5-fpm
- php5-memcache
- php5-memcached
- php5-xdebug
- phpunit
```

**Comment by zach-adams** [comment]
Good idea! I also noticed it's missing php5-curl which is another important PHP package

**Comment by ruudvh** [comment]
Great, thank you!
Fixed in a89b2592851811fa6db785634ac1a468ea7fb132
